assets/src/block-editor/helpers/index.js

```diff
@@ -386,17 +386,32 @@ const AmpLayoutControl = ( props ) => {
 	}
 
 	return (
-		<SelectControl
-			label={ label }
-			value={ ampLayout }
-			options={ getLayoutOptions( name ) }
-			onChange={ ( value ) => {
-				setAttributes( { ampLayout: value } );
-				if ( 'core/image' === props.name ) {
-					setImageBlockLayoutAttributes( props, value );
-				}
-			} }
-		/>
+		<>
+			<Notice
+				status="warning"
+				isDismissible={ false }
+			>
+				<span dangerouslySetInnerHTML={ {
+					__html: sprintf(
+						/* translators: placeholder is link to support forum. */
+						__( 'The AMP Layout setting is deprecated and is slated for removal. Please <a href="%s" target="_blank" rel="noreferrer">report</a> if you need it.', 'amp' ),
```

Comment:
This seems to be copy/pasta from #5573. Shouldn't it reference “AMP Noloading” and be put on `AmpNoloadingToggle` below instead?

Reply:
Yes it should :facepalm:. Resolved in https://github.com/ampproject/amp-wp/pull/5574/commits/d8a4e0cc238821d790c6e5c2c2c292e9bfb8ca3b.